google/cloud/bigquery/query.py

```diff
@@ -325,35 +332,46 @@ class ScalarQueryParameter(_AbstractQueryParameter):
     """Named / positional query parameters for scalar values.
 
     Args:
-        name (Optional[str]):
```

Comment:
With the type annotations in the actual method signature, these are unnecessary.

<img width="849" alt="Screen Shot 2021-08-03 at 3 46 15 PM" src="https://user-images.githubusercontent.com/247555/128084026-cc1bc8ac-27c7-4701-8198-9953a32039b1.png">